javascript/ql/src/semmle/javascript/ApiGraphs.qll

```diff
@@ -384,6 +381,8 @@ module API {
         imports(_, m)
         or
         m = any(CanonicalName n | isUsed(n)).getExternalModuleName()
+        or
+        any(TypeAnnotation n).hasQualifiedName(m, _)
```

Comment:
Do we want to add `or any(Type t).hasUnderlyingType(m, _)` here as well?

Reply:
Yes we do. And the remaining references to `CannonicalName` can be deleted.  
https://github.com/github/codeql/pull/5460